### PR TITLE
Update boilerplate

### PR DESCRIPTION
**Pull request by MaudSimon**

### CODE REVIEW

File: src/content/homepage.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 headerSection:
   dark:
     title: Blockchain v3.0
-    subtitle: Alephium is the first operational sharded blockchain bringing scalability,
-      ETH-inspired smart contracts and dApps capabilities to Bitcoin's proven core
-      technologies while ensuring better performance and improved energy efficiency.
+    subtitle: Alephium is the first live layer 1 sharded blockchain scaling 
+    and improving on Bitcoin core technologies, Proof of Work & UTXO. It delivers
```
Comment (nop33) [inline src/content/homepage.md]: - I would replace _"...Proof of Work & UTXO"_ with _"...Proof of Work, and UTXO"_.

File: src/content/homepage.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 headerSection:
   dark:
     title: Blockchain v3.0
-    subtitle: Alephium is the first operational sharded blockchain bringing scalability,
-      ETH-inspired smart contracts and dApps capabilities to Bitcoin's proven core
-      technologies while ensuring better performance and improved energy efficiency.
+    subtitle: Alephium is the first live layer 1 sharded blockchain scaling 
+    and improving on Bitcoin core technologies, Proof of Work & UTXO. It delivers 
+    a highly performant, secure DeFi & Dapps platform with enhanced energy efficiency.
```
Comment (nop33) [inline src/content/homepage.md]: - Same regarding the "&"
- Is the format of the word "Dapp" the correct one? Not "dApp" or "dapp" anywhere else in our copy?

Comment (nop33) [reply src/content/homepage.md]: I can find many places in our homepage where we say "dApp".

Comment (MaudSimon) [reply src/content/homepage.md]: you are correct, it should dApp!

File: src/content/homepage.md (hunk 1 above)
Comment (MaudSimon) [reply src/content/homepage.md]: fine by me except the "," after pow

**Pull request merged**